src/login/71-seat.rules.in

```diff
@@ -13,6 +13,10 @@ TAG=="uaccess", SUBSYSTEM!="sound", TAG+="seat"
 SUBSYSTEM=="sound", KERNEL=="card*", TAG+="seat"
 SUBSYSTEM=="input", KERNEL=="input*", TAG+="seat"
 SUBSYSTEM=="graphics", KERNEL=="fb[0-9]*", TAG+="seat"
+
+# HyperV requires master-of-seat to be set
```

Comment:
this comment doesn't really help. I mean, the line itself already says exactly that, no, otherwise you wouldn't have added that...

Please say something like "HyperV currently doesn't do DRM, hence we need to synthesize for HyperV's fb device instead"